readme.md

```diff
@@ -394,16 +394,23 @@ entities:
     name: living temperature in Farenheit # Overrides the entity name
     unit_of_measurement: °F # Overrides the unit
     show_value: true # shows the last value as text
+    customdata: |
+      $fn ({states}) => 
+        states.map( () => ({ extra_attr: "hello" }) )
```

Comment:
I meant a specific attribute instead of all of them. That way the reader can make the connection (instead of adding all attributes with the spread operator and having them "magically appear"). This is mostly because many users of this card dont know much js :)

Reply:
As we have `%{y}` and ability to get value of `name` or `unit_of_measurement` via `getConfig`, I cannot think of a good usage for custom attribute anymore...

Maybe like this?

``` yaml
    customdata: |
      $fn ({states}) => 
        states.map( ({state}) => ({ value_with_suffix: state + " W" }) )
    texttemplate: >- # custom format for show_value
      <b>%{customdata.value_with_suffix}</b><br>
```

Reply:
Oh, I see what you mean. I had entities with extra attributes in mind. Like a climate entity.

Reply:
I would maybe remove example for `customdata` completely and just recommend using `getFromConfig` - what do you think?

Reply:
Ok, sounds good!
If you want to make it super compact, you could use "get" (is just an alias of getFromConfig) and $ex instead of $fn (it's the same except one only needs the body of the function, so no parameters and arrow)

Reply:
I'll think about it and issue a PR.